core/store/ledgerstore/event_store.go

```diff
@@ -173,9 +166,6 @@ func (this *EventStore) GetCurrentBlock() (common.Uint256, uint32, error) {
 	key := this.getCurrentBlockKey()
 	data, err := this.store.Get(key)
 	if err != nil {
-		if err == leveldb.ErrNotFound {
-			return common.Uint256{}, 0, nil
-		}
 		return common.Uint256{}, 0, err
```

Comment:
common.UINT256_EMPTY ?
I saw you changed Uint256{} to common.UINT256_EMPTY in some other places.

Reply:
Uint256{} the same with UINT256_EMPTY